tools/benchmark.py

```diff
@@ -341,6 +342,12 @@ def process_events_thread(watcher, data, lock):
                         data[e_name]["signed_at"] = now()
                     data[e_name]["signed"] = annotations["chains.tekton.dev/signed"]
 
+            # Determine terminating status
+            if "deletionTimestamp" in data[e_name]:
+                data[e_name]["terminated"] = True
+            else:
+                data[e_name]["terminated"] = False
+
```

Comment:
I think we would also use something like:

            # Determine terminating status
            if event["type"] == "DELETED":
                data[e_name]["deleted"] = True
                if "deleted_at" not in data[e_name]:
                    data[e_name]["deleted_at"] = now()
            else:
                data[e_name]["deleted"] = False

Reply:
+ relevant column below

Reply:
Updated to "DELETED" event type and column names to "deleted" in the output.

Reply:
Now, thinking about it `if "deleted_at" not in data[e_name]:` is never `False` right? :) Anyway, not a big deal. Thank you!